src/vivarium_gates_nutrition_optimization/components/hemoglobin.py

```diff
@@ -298,7 +298,7 @@ def setup(self, builder: Builder):
         )
 
         builder.value.register_value_modifier(
-            "disability_weight",
+            "all_causes.disability_weight",
```

Comment:
This is a pretty important difference! "all_causes" is not set up as a SimpleCause (on the backend in vph) and so it now has a `state_id`. So when you need access to the all causes dw pipeline, it's now called `all_causes.disability_weight` as opposed to just `disability_weight`.